server/sonar-process-monitor/src/main/java/org/sonar/application/config/JdbcSettings.java

```diff
@@ -119,23 +126,23 @@ void checkUrlParameters(Provider provider, String url) {
     }
   }
 
-  private static void warnIfUrlIsSet(String port, String existing, String expectedUrl) {
+  private static void warnIfUrlIsSet(int port, String existing, String expectedUrl) {
     if (isNotEmpty(existing)) {
       Logger logger = LoggerFactory.getLogger(JdbcSettings.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1312)